resources/core/charts/service-catalog/charts/catalog/values.yaml

```diff
@@ -66,11 +66,20 @@ apiserver:
     type: etcd
     # Further configuration for the etcd-based backend
     etcd:
+      # Whether to enable TLS communitation with etcd
+      tls:
+        enabled: true # Remember to switch proto (http/https) in servers url accordingly
+
+        ## If etcd tls is enabled you need to provide name of secret which stores 3 keys:
+        ## etcd-client-ca.crt - SSL Certificate Authority file used to secure etcd communication
+        ## etcd-client.crt - SSL certification file used to secure etcd communication.
+        ## etcd-client.key - SSL key file used to secure etcd communication.
+        clientCertSecretName: core-service-catalog-etcd-etcd-client-tls
       # Whether to embed an etcd container in the apiserver pod
       # THIS IS INADEQUATE FOR PRODUCTION USE!
       useEmbedded: false
       # etcd URL(s); override this if NOT using embedded etcd
-      servers: http://core-service-catalog-etcd-client.kyma-system.svc.cluster.local:2379
+      servers: https://core-service-catalog-etcd-client.kyma-system.svc.cluster.local:2379
```

Comment:
this will also work it the TLS is disabled? because in `backup-job.yaml` you have
```
            {{- if .Values.cluster.TLS.enabled }}
              value: https://core-service-catalog-etcd-client.kyma-system.svc.cluster.local:2379
            {{ else }}
              value: http://core-service-catalog-etcd-client.kyma-system.svc.cluster.local:2379
            {{ end }}
```

Reply:
to enable/disable tls you need to change 3 values:
- in etcd subchart: cluster.TLS.enabled (true/false)
- in catalog subchart:
  - apiserver.storage.etcd.tls.enabled (true/false)
  - apiserver.storage.etcd.servers (http/https)